mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
I was thinking of an API like this:

```py
raw = mne.io.read_raw_eeglab(fname)
events, event_id = read_events_eeglab(fname)
epochs = mne.Epochs(raw, events, event_id)
```
not sure if we're talking about the same thing :)